frontend/src/libs/recorder.js

```diff
@@ -0,0 +1,318 @@
+'use strict';
+
+Object.defineProperty(exports, '__esModule', {
+  value: true,
+});
+exports.Recorder = undefined;
```

Comment:
This seems to be pre-built babelized code. Where does this code come from? Is this your code?
Also, *everything* needs copyright headers.

Reply:
@greentfrapp do u know where this is from?

Reply:
This comes from here: https://github.com/mattdiamond/Recorderjs
But I probably should have gotten it from dist instead of src. Didn't use yarn add because the current recorder-js is by a different developer and is more cumbersome to use. Will update this.

Reply:
That library says:
> Note: This repository is not being actively maintained due to lack of time and interest. If you maintain or know of a good fork, please let me know so I can direct future visitors to it. In the meantime, if this library isn't working, you can find a list of popular forks here: http://forked.yannick.io/mattdiamond/recorderjs.

This is not great. In any case, if you copy any code from anywhere you must include copyright notices.

Reply:
Okay, will add copyright notice and try to find a more updated version.